docs/dev/adding_instruments.rst

```diff
@@ -187,28 +187,6 @@ In addition, any entries in ``**kwargs**`` take precedence, so if they need to,
 
 For many instruments, the simple way presented first is enough, but in case you have a more complex arrangement to implement, pymeasure has your back!
```

Comment:
Seeing that we delete the originally following section, where should we point to, instead, if people have "a more complex arrangement"? To the "advanced" section below?

Reply:
Yes, as that is exactly the place, where we talk about more complex communication.

Reply:
Perfect!